Makefile

```diff
@@ -1,20 +1,15 @@
 #
 # This mostly contains shortcut for multi-command steps.
 #
-SRC = india_forecast_app tests
+SRC = india_forecast_app scripts tests
 
 .PHONY: lint
 lint:
 	poetry run ruff $(SRC)
-	poetry run black --check $(SRC)
-	poetry run isort --check $(SRC)
```

Comment:
do you want to keep this in? I have found them quite useful in the past

Reply:
I was getting a conflict between ruff and black, so just stuck with ruff - as far as I understand, ruff is a drop in replacement for black anyway